### PR TITLE
Add Github Actions CI.

### PR DESCRIPTION
**Pull request by inferno-chromium**
Fixes https://github.com/google/oss-fuzz/issues/3970

### CODE REVIEW

File: .github/workflows/project_tests.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    steps:
+    - uses: actions/checkout@v2
+    - run: |  # Needed for git diff to work.
+        git fetch origin master --depth 1
+        git symbolic-ref refs/remotes/origin/HEAD refs/remotes/origin/master
+
+    - name: Clear unnecessary files
+      run: |
+        sudo swapoff -a
```
Comment (oliverchang) [inline .github/workflows/project_tests.yml:47]: these seem pretty hacky. Are they really needed? e.g. do we need "apt clean" if we don't install any apt packages?

Comment (inferno-chromium) [reply .github/workflows/project_tests.yml:47]: This is directly coming as a recommendation from GitHub Actions support channel (we use this in FuzzBench as well). See https://github.community/t/bug-strange-no-space-left-on-device-ioexceptions-on-github-runners/17616/9

**Comment by inferno-chromium** [comment]
Amazing ffmpeg survived with all sanitizers on CI!

**Comment by jonathanmetzman** [comment]
> Amazing ffmpeg survived with all sanitizers on CI!

Indeed. Very impressive.

**Pull request merged**